to-do-app-secrets.yaml

```diff
@@ -0,0 +1,19 @@
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+stringData:
+  database-name: kamolsahamongo  
+  
+  mongo-db-connection: mongodb://kamolsahamongo:NeLEHTV8Tmny6lpffYAHrZOiDm8gceLz0iLqlj5WwpTUZqU0mp3lUCwJKF2EDqdsaUl9fK0rWGfl40ZobliAAQ==@kamolsahamongo.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@kamolsahamongo@
+
+  secret-key: secret-key
+
+  client-id: a40204a3c886e5b6441d
+
+  client-secret: 823f2dd80b2ae2dd65aee9f3c4d1f99becb49b72
+  
+  port: "80"
+
```

Comment:
You've leaked all your credentials. Everyone has access to all of these things that are meant to be secret.

Secrets should never be committed to Git. You should remove this file from GitHub and in a real scenario it would be important to also rotate all the secrets because they have been compromised. But for this exercise, we don't especially care if someone can spoof your app or read your database. We'll be tearing down all the azure resources soon anyway.